src/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp

```diff
@@ -53,8 +55,8 @@ void ConvolutionTransformation::SetUp() {
         param.fakeQuantizeOnWeights);
 }
 
-void ConvolutionTransformation::Run() {
-    LayerTestsCommon::Run();
+void ConvolutionTransformation::run() {
+    LayerTransformation::run();
```

Comment:
Open question: Replace by new namespace?? `ov::Transformation` or something else? Where is method definition?

Reply:
it's not namespace. `LayerTransformation` is base class for all LPT functional tests.